e2e/drift/drift_test.go

```diff
@@ -175,7 +179,9 @@ var _ = Describe("Drift", func() {
 				Eventually(func() string {
 					out, _ := k.Namespace(env.Namespace).Get("bundles", bundleName, "-o=jsonpath={.status.conditions[*].message}")
 					return out
-				}).Should(ContainSubstring(`error reconciling drift: cannot patch "nginx-service" with kind Service: Service "nginx-service" is invalid: spec.ports[1].name: Duplicate value: "http"]; service.v1 drift/nginx-service modified {"spec":{"ports":[{"name":"http","port":80,"protocol":"TCP","targetPort":"http-web-svc"},{"name":"http","port":1234,"protocol":"TCP","targetPort":"http-web-svc"}]}}`))
```

Comment:
I think it was better to avoid concatenation, it makes it easier to copy&paste.

Reply:
I see what you mean and agree on that point, but shorter, more readable lines make it easier to see the diff/drift that we expect in the condition message here.